.github/workflows/testbuild.yml

```diff
@@ -38,11 +44,19 @@ jobs:
           echo "Building for $CHANGED_LOCATIONS"
           echo "CHANGED_LOCATIONS=$CHANGED_LOCATIONS" >> "$GITHUB_ENV"
 
-      - name: Build changed locations
+      - if: ${{ github.event_name == 'pull_request' }}
```

Comment:
this should still happen on pull_request and push as we still have this one:

![image](https://github.com/freifunk-berlin/bbb-configs/assets/198902/9c1af706-5408-4b1b-8ee7-64cec62289f9)

Reply:
The original code states
```
name: Test Build
on: [pull_request]
```
so it was running only on PRs. But as we discussed this bypass before i would just go and remove it.